R/constants.R

```diff
@@ -247,6 +247,9 @@ ELECTRICITY_INPUT_FUELS<- c("biomass", "coal", "gas", "refined liquids")
 energy.CLIMATE_NORMAL_YEARS <- 1981:2000
 energy.RSRC_FUELS <- c("coal", "gas", "refined liquids")
 
+#Assumed base year heat price, used for calculating adjustment to non-energy costs of electricity technologies with secondary output of heat
```

Comment:
Space after `#`; please add units for both constants, and prefix their names with `energy.`